infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/LogReplicationConfig.java

```diff
@@ -37,9 +37,9 @@ public class LogReplicationConfig {
     // Default value for the max number of entries applied in a single transaction on Sink during snapshot sync
     public static final int DEFAULT_MAX_SNAPSHOT_ENTRIES_APPLIED = 50;
 
-    // Max message size supported by protocol buffers is 64MB. Log Replication uses this limit as the default message
+    // Max message size supported by protocol buffers is 25MB. Log Replication uses this limit as the default message
     // size to batch and send data across over to the other side.
-    public static final int MAX_DATA_MSG_SIZE_SUPPORTED = (64 << 20);
+    public static final int MAX_DATA_MSG_SIZE_SUPPORTED = 25 * 1_024 * 1_024;
```

Comment:
is this a typo?

Reply:
No this is intentional, same thing as bit shifting left by 20 but felt this was more indicative of the size in bytes